x-pack/plugins/index_management/public/application/components/component_templates/component_template_selector/component_templates_list_item.scss

```
@@ -8,7 +8,7 @@
   &--selected {
     &::before {
       content: '';
-      background-color: rgba(255, 255, 255, .7);
+      background-color: rgba($euiPageBackgroundColor, .7);
```

Similar comment here. Would `$euiColorEmptyShade` be a better color variable choice (as it is the same color as `255, 255, 255` in light mode, but will correctly change colors for dark mode)?

```suggestion
      background-color: rgba($euiColorEmptyShade, .7);
```